src/style.css

```diff
@@ -1,3 +1,85 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  margin: 0;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-sizing: border-box;
+  width: 100%;
+  align-items: center;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav_item {
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+  margin: 0 0 0 20px;
+}
+
+.nav_item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav_link {
+  text-decoration: none;
+  display: flex;
+  text-transform: uppercase;
+  color: #000;
+  font-size: 12px;
+  white-space: nowrap;
+  font-weight: 500;
+  line-height: 60px;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+  font-style: normal;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+/* [data-qa='hover'] {
+  color: #00acdc;
+} */
+ 
+:hover {
+  color: #00acdc;
+}
+
+.is-active::after {
+  height: 4px;
+  width: 100%;
+  content: '';
+  background-color: #00acdc;
+  position: absolute;
+  bottom: 0;
+  border-radius: 8px;
+  left: 0;
+}
```

Comment:
You have used the `::after` pseudo-element to add a visual effect to your `.is-active` class. This is a great use of pseudo-elements! Just remember that pseudo-elements are not part of the DOM and can't be styled as regular elements. But in your case, it's used correctly.